HopsanGUI/Utilities/GUIUtilities.cpp

```diff
@@ -688,8 +688,14 @@ double firstTimeAt(QVector<double> vData, QVector<double> vTime, double value)
 //! @param sigma Standard deviation of normal distribution
 double normalDistribution(double average, double sigma)
 {
-    double U1 = (double)rand() / (double)RAND_MAX;
-    double U2 = (double)rand() / (double)RAND_MAX;
+    double U1 = 0;
+    double U2 = 0;
+    while(U1 == 0) {
+        U1 = (double)rand() / (double)RAND_MAX;
+    }
+    while(U2 == 0) {
```

Comment:
OK U1 should not be zero, but why not allow U2 to be zero? not sure if it matters.

Reply:
You are right, I will fix it. The equivalent function in HopsanCore only checked U1.